Doc/reference/expressions.rst

```diff
@@ -1741,6 +1741,11 @@ returns a boolean value regardless of the type of its argument
 (for example, ``not 'foo'`` produces ``False`` rather than ``''``.)
 
 
+.. index::
+  single: := (colon equals)
+  single: assignment expression
+  single: walrus operator
+  single: named expression
 Assignment expressions
```

Comment:
Fixed indentation and added missing newline:
```suggestion
.. index::
   single: := (colon equals)
   single: assignment expression
   single: walrus operator
   single: named expression

Assignment expressions
```

Reply:
Thanks!